packages/common/src/api/tan-query/types.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Standard tan-query pass-thru options that we use
+ */
+export type QueryOptions = {
```

Comment:
nit: can/should pick these from the types exported from tan-query if they are really passthroughs.